server/src/operators/chunk_operator.rs

```diff
@@ -457,7 +566,7 @@ pub async fn insert_chunk_metadata_query(
     Ok(chunk_data)
 }
 
-// Reverts insertion event, DO NOT USE IF UPSERT == TRUE
+/// Bulk revert, assumes upsert chunk_ids were not upserted, only enterted
```

Comment:
What happens to the chunks if it gets reverted?

Reply:
It deletes the chunks it created, the reason why is because we have retries, if it retires to add it again it will fail because of `chunk_id`, `group_id`, and `file_id` collision.